CMakeLists.txt

```diff
@@ -291,6 +291,16 @@ endmacro (tests_hook)
 # all setup common to the OPM library modules is done here
 include (OpmLibMain)
 
+if(MPI_FOUND AND HDF5_FOUND AND NOT HDF5_IS_PARALLEL)
+  message(WARNING "When building parallel OPM flow we need a "
+    "parallel version of hdf5, but found only a serial one. "
+    "Please install a parallel hdf5 library for MPI "
+    "(e.g with apt-get install libhdf5-mpi-dev) and do a clean "
+    "rebuild (build after \"make clean\". Continuing without"
+    "hdf5 and parallel restart.")
```

Comment:
Technically we still have the original restart files, so maybe we can say
```
Disabling HDF5-based restart files
```
or something along those lines to avoid confusion?  In any case there should be a space after "without" and before "hdf5".  You should probably close the parentheses following "(build after..." too.

Reply:
Changed to "Continuing with only normal restart (not hdf5 enabled one)"